src/admin/components/driver/DriverInfo.tsx

```diff
@@ -0,0 +1,205 @@
+import { useParams } from "react-router-dom";
+
+import toast from "react-hot-toast";
+import { useEffect, useState } from "react";
+import { useRepository } from "../../../hooks/CustomHook";
+import { capitalize } from "../../../utils/utilities";
+export default function DriverInfo() {
+  const [open, setOpen] = useState<boolean>(false);
+  const [report, setReport] = useState<string>("");
+  const { id } = useParams();
+  const { repo } = useRepository();
+  function handleSendMessage() {
+    if (confirm("Are you sure!") == true) {
```

Comment:
Let's use lodash package's "isEqual" method to check equality whenever necessary.